test/test-applications/integrations/TestApplication.StackExchangeRedis/Properties/launchSettings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "profiles": {
+    "TestApplication.StackExchangeRedis": {
+      "commandName": "Project",
+      "commandLineArgs": "--mongodb 5295",
```

Comment:
We can probably clean up all of the mongodb references from this file, including the additional sources variable.

Reply:
The file was included by mistake. Removed.